README.md

```diff
@@ -1 +1,127 @@
-# angular-behavioural-directives-readme
\ No newline at end of file
+# Behavioural Directives
+
+## Overview
+
+Now we've learnt event directives, we can move onto the equally, if not more, powerful behavioural directives.
+
+## Objectives
+
+- Describe behavioural Directives
+- Use ng-repeat
+- Use ng-model on an input
+- Interpolate ng-model result in the View
+- Conditionally add classes with ng-class
+
+## What's a behavioural directive?
+
+Behavioural directives are directives that allow us to manipulate the DOM. For example, we could have a repeating list of items or we could hide/show a DOM node depending on a variables value.
+
+Behavioural directives are commonly just used as attributes, and quite a lot are provided by Angular itself.
+
+Imagine we're not using Angular. If we wanted to loop through an array of objects and display them, we'd have to do something like this:
+
+
+```js
+var todos = [{
+  title: 'Learn Angular',
+  complete: false,
+},{
+  title: 'Create GitHub profile',
+  complete: false
+},{
+  title: 'Brush teeth',
+  complete: true
+}];
+
+var list = document.querySelector('ul.todos');
+
+todos.forEach(function (todo) {
+  var item = document.createElement('li');
+  
+  var wording = todo.complete ? 'Completed!' : 'Not completed :(';
+  
+  item.innerHTML = '<h4>' + todo.title + '</h4> ' + wording;
+  
+  list.appendChild(item);
+});
+```
+
+This worked fine but what if our `todos` array was to get updated? We'd have to make sure to only insert the new items into our list, and only remove the correct elements when an item was removed from our array. This could get extremely messy and out of sync with our data if we were not really careful.
+
+## Angular saves the day!
+
+Try and contain your excitement when you see how this is done in Angular.
+
+The HTML:
+
+```html
+<ul>
+  <li ng-repeat="todo in vm.todos">
+    <h4>{{ todo.title }}</h4>
+    
+    {{ todo.complete && 'Completed!' || 'Not completed :(' }}
+  </li>
+</ul>
+```
+
+Our controller: 
+
+```js
+function TodosController() {
+  this.todos = [{
+     title: 'Learn Angular',
+     complete: false,
+   },{
+     title: 'Create GitHub profile',
+     complete: false
+   },{
+     title: 'Brush teeth',
+     complete: true
+   }];
+}
+```
+
```

Comment:
Here, can you break down the above example a bit? What are the different pieces that make this come together? Start with `ng-repeat="todo in vm.todos"` - what is that giving to us? What does `{{ todo.complete && 'Completed!' || 'Not completed :(' }}` do for us? Walk us through it a little bit after showing us the code.